subsystems/motor/ServoSubsystem.java

```diff
@@ -70,7 +70,7 @@ public ServoSubsystem(Servo... servos) {
 		this("ServoSubsystem", servos);
 	}
 
-	@Override
+
```

Comment:
This seems a little concerning but I'm not actually sure if its wrong. Is there a reason that this can no longer override anything, and are you sure there isn't an alternate method you are supposed to override instead?

Reply:
This was sort-of a last resort fix, we looked for a while but couldn't find an actual solution (such as a method to replace it with). I knew this wasn't the best fix, but it was the only thing we could find to fix the error, other than completely removing it.

Reply:
what we are going with is completely removing `initDefaultCommand()`s and adding javadoc comments to the classes that they are from saying what the default command should be set to.